room/runtime/src/main/java/androidx/room/SQLiteCopyOpenHelper.java

```diff
@@ -63,9 +66,21 @@
             @Nullable File copyFromFile,
             int databaseVersion,
             @NonNull SupportSQLiteOpenHelper supportSQLiteOpenHelper) {
+        this(context, copyFromAssetPath, copyFromFile, false, databaseVersion,
```

Comment:
kept this method for binary compatibility purposes - but now that I think about, I guess this class is not open for the public anyway so I can probably remove this constructor

Reply:
(removed).

Reply:
sometimes we need to them if generated code calls it but otherwise, we can just update. i think generated code is not calling here so should be fine, i'll defer to @danysantiago .

Reply:
Should be fine to update this constructor, not called by gen code.